### PR TITLE
Update Omniauth settings

### PR DESCRIPTION
**Pull request by schgressive**
Set to true to use https for the avatar image url returned in the auth hash.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -220,9 +220,9 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :facebook, ENV['FB_APP_ID'], ENV['FB_APP_SECRET'], scope: 'email, user_about_me, publish_actions', image_size: "large"
-  config.omniauth :twitter, ENV['TW_CONSUMER_KEY'], ENV['TW_CONSUMER_SECRET'], image_size: "original" # 73 px
-  config.omniauth :google_oauth2, ENV['GOOGLE_KEY'], ENV['GOOGLE_SECRET'], image_size: {width: 200, height: 200}
+  config.omniauth :facebook, ENV['FB_APP_ID'], ENV['FB_APP_SECRET'], scope: 'email, user_about_me, publish_actions', image_size: "large", secure_image_url: truel
```
Comment (emilioeduardob) [inline config/initializers/devise.rb]: thanks @schgressive , I believe this would be true not `truel` ;)

Comment (schgressive) [inline config/initializers/devise.rb]: lol

Comment (zevarito) [inline config/initializers/devise.rb]: Why twitter and google_oauth2 are removed?

**Comment by schgressive** [comment]
only in facebook and twitter, add secure_image_url: true, google is fine by default is https

**Comment by emilioeduardob** [comment]
I fixed the typo and merged to master. Thanks

**Pull request closed without merging**